css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - flex-wrap</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+        padding: 1rem;
+      }
+
+      .card-wrapper {
+        flex-basis: calc(100% / 6);
+        flex-direction: row;
```

Comment:
The `.card-wrapper` element is not a flex container, so this property is doing nothing, it can be removed.